Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM registry.redhat.io/openshift4/ose-tools-rhel8:v4.13.0-202306070816.p0.g05d83ef.assembly.stream
```

Comment:
On the Red Hat catalog, I see this as the latest release which was published a month:
https://catalog.redhat.com/software/containers/openshift4/ose-cli/5cd9ba3f5a13467289f4d51d?tag=v4.13.0-202305291355.p0.g1024efc.assembly.stream&push_date=1686102635000&container-tabs=overview

As consumers of that image, I wonder if we shouldn't use the image published on the Red Hat catalog ?

Reply:
it the same image, we're just using the url that does not require authentication. It could be found on the catalog as well.

Reply:
Didn't you mention in the `README.md` that it's required to login to the registry in order to access the base image?

Reply:
Actually yeah, `registry.redhat.io` does require a login, and I am using what seem to be the latest, most recent version in the catalog (Latest build I can find of 4.13.0), so not sure what you are asking.

Reply:
Ok there seems to be come mess on the catalog site, clicking on the link to the Jul 6th image I'm using prongs you to the May 29th image you linked to....